### ISSUE TITLE
Unnecessary horizontal and vertical scrollbars included by default in HTML R Markdown

### ISSUE THREAD

**Issue by Michael-Sheppard** [davidgohel/flextable#439]
After recent updates to the package, it seems that horizontal and vertical scrollbars are always added by default in an HTML R Markdown document even if they aren't necessary. I cannot find an obvious way to prevent the scrollbars from appearing. 

Example R Markdown code: 

```
---
title: "Example of `flextable` Scrollbars"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
library(tidyverse)
library(flextable)
```
```{r cars}

cars %>%
  flextable() %>%
  autofit() %>%
  theme_box() 

```
```

**Comment by davidgohel** [comment]
Could you add `sessionInfo()`?

**Comment by Michael-Sheppard** [comment]
This was resolved after updating from version 0.7.2 to 0.7.3. Issue can be closed.
